projects/js-packages/components/components/dialog/index.tsx

```diff
@@ -35,14 +41,10 @@ export default function Dialog( { primary, secondary, split } ) {
 			</Col>
 		</Container>
 	);
-}
-
-Dialog.propTypes = {
-	split: PropTypes.bool,
 };
 
 Dialog.defaultProps = {
-	primary: null,
-	secondary: null,
 	split: false,
 };
```

Comment:
I think we can remove `defaultProps` and directly use the default values via ES6 default params.